pom.xml

```diff
@@ -109,6 +109,7 @@
         <mockito-core.version>2.8.47</mockito-core.version>
         <hamcrest-all.version>1.3</hamcrest-all.version>
         <hamcrest-junit.version>2.0.0.0</hamcrest-junit.version>
+	<hamcrest-optional.version>1.0</hamcrest-optional.version>
```

Comment:
✏️  Indentation seems off.

Reply:
tilt your head a bit, it is perfectly straight 😆 
I hate the fact that we no longer have a formatter for these file and the one we are using in IntelliJ seems to mess up every files except .java ones 😞